src/Tempest/Framework/Testing/Http/TestResponseHelper.php

```diff
@@ -219,6 +219,78 @@ public function assertNotSee(string $search): self
         return $this;
     }
 
+    public function assertViewData(string $key, ?Closure $test = null): self
```

Comment:
Let's call this `$callback` for consistency. Can you also add docblocks to the closure so users can know what arguments it takes?

```suggestion
    public function assertViewData(string $key, ?Closure $callback = null): self
```